src/ServiceControl.AcceptanceTests/MessageFailures/When_an_event_with_multiple_subscribers_fails.cs

```diff
@@ -0,0 +1,131 @@
+﻿namespace ServiceBus.Management.AcceptanceTests.MessageFailures
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using NServiceBus;
+    using NServiceBus.AcceptanceTesting;
+    using NServiceBus.Config;
+    using NServiceBus.Features;
+    using NUnit.Framework;
+    using ServiceBus.Management.AcceptanceTests.Contexts;
+    using ServiceControl.MessageFailures.Api;
+
+    class When_an_event_with_multiple_subscribers_fails : AcceptanceTest
+    {
+        [Test]
+        public void There_should_be_a_FailedMessage_for_each_subscriber()
+        {
+            var context = new FailingEventContext();
+
+            var failedMessages = new List<FailedMessageView>();
+
+            Define(context)
+                .WithEndpoint<FailingSubscriber1>(behavior => behavior.Given((bus, ctx) =>
+                {
+                    if (ctx.HasNativePubSubSupport)
+                    {
+                        ctx.Subscriber1Subscribed = true;
+                    }
+                }))
+                .WithEndpoint<FailingSubscriber2>(behavior => behavior.Given((bus, ctx) =>
+                {
+                    if (ctx.HasNativePubSubSupport)
+                    {
+                        ctx.Subscriber2Subscribed = true;
+                    }
+                }))
+                .WithEndpoint<Publisher>(behavior => behavior
+                        .CustomConfig(cfg => cfg.OnEndpointSubscribed(sub =>
+                            {
+                                if (sub.SubscriberReturnAddress.Queue.Contains("Subscriber1"))
+                                {
+                                    context.Subscriber1Subscribed = true;
+                                }
+                                else if (sub.SubscriberReturnAddress.Queue.Contains("Subscriber2"))
+                                {
+                                    context.Subscriber2Subscribed = true;
+                                }
+                                else
+                                {
+                                    throw new Exception($"Unknown subscriber subscribed to publisher: {sub.SubscriberReturnAddress}");
+                                }
+                            })
+                        ).When(
+                            ctx => ctx.Subscriber1Subscribed && ctx.Subscriber2Subscribed,
+                            bus => bus.Publish<SampleEvent>()
+                        )
+                ).Done(ctx => TryGetMany("/api/errors", out failedMessages) && failedMessages.Count > 1)
```

Comment:
nitpicking: maybe better instead of `count > 1` use `count >= 2` to express that we expect at least two.